src/blocks/business-hours/index.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+import { registerBlockType } from '@wordpress/blocks';
+
+import { Placeholder } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import './editor.scss';
+import { mapIcon as icon } from '../../helpers/icons.js';
```

Comment:
Need to find a better icon.

Reply:
@HardeepAsrani, any idea from where we can get one?

Reply:
Either you can pick one from here: https://wordpress.github.io/gutenberg/?path=/story/icons-icon--library

Or here: https://themeisle.com/free-icons/